In [1]:
n1=100
n2=200
n1
n2

200

In [2]:
# if you want to see both n1 and n2 use Print()
n1=100
n2=200
print(n1)
print(n2)

100
200


In [4]:
print("helllo")
print("how")
print("are")
print("you")
print(hello) # python will give error in this line , because it is step by step process
print(n1)  # if u gat error in 5 th line ,6 th onwards won't execuite

helllo
how
are
you


NameError: name 'hello' is not defined

In [5]:
print(n1,n2)

100 200


$end$ $operator$

In [1]:
print("hi", end=' ')
print("how")
print("are")

hi how
are


In [15]:
print("hi........")
print("how", end=' ')
print("are", end=' ')
print("you",end='----------?')

hi........
how are you----------?

In [23]:
num1=500
num2=200
add=num1+num2
print("the addition of",num1,"and",num2,"is=",add) # this , and " are kichoidi so we can use f"content{name}" like this we can 

the addition of 500 and 200 is= 700


In [27]:
name = input("Enter your name")
city = input("Enter your city")
age = input("Enter your age")
print(f"my name is {name} , I came from {city}  and Im {age} years old")

Enter your name siddesh
Enter your city chitradurga
Enter your age 22


my name is siddesh , I came from chitradurga  and Im 22 years old


In [28]:
# formate we can write like this also
name = input("Enter your name")
city = input("Enter your city")
age = input("Enter your age")
print("my name is {} , I came from {}  and Im {} years old".format(name,city,age))

Enter your name siddesh
Enter your city chitradurga
Enter your age 22


my name is siddesh , I came from chitradurga  and Im 22 years old


In [34]:
val1=20
val2=30
add=val1+val2
sub=val1-val2
mul=val1*val2
div=round(val1/val2,2)   # important round off used
print(f"addition {val1} and {val2} is ",add)
print(f"subtraction {val1} and {val2} is ",sub)
print(f"Multiplication {val1} and {val2} is ",mul)
print(f"division {val1} and {val2} is ",div)

addition 20 and 30 is  50
subtraction 20 and 30 is  -10
Multiplication 20 and 30 is  600
division 20 and 30 is  0.67


In [36]:
print(5/2)  # division
print(5//2)   # floor division =quotient
print(5%2)   # modulus = remainder

2.5
2
1


In [41]:
# taking input from user by using "input"
v1=int(input("enter value 1"))          # if we take int(input("   ")) then it will take it as integer other wise it will consider as string
v2=int(input("enter value 2"))             # if only use input("  ")  it takes input as string
add= v1 + v2
print(f"addition {v1} and {v2} is ",add)

enter value 1 10
enter value 2 20


addition 10 and 20 is  30


30

In [45]:
num1=int(input("enter value 1"))         
num2=float(input("enter value 2")) 
add=num1 + num2
print(f"addition {num1} and {num2} is ",add)

enter value 1 100
enter value 2 200.5


addition 100 and 200.5 is  300.5


- int("200") will work
- int("200.5") will not work
- float("100") will work
- float("100.5") will work    
-  some data type will wor some of them will not work for this reason we will use "EVAl"
-  "" eval "" it is only for numbers 

$eval$

In [48]:
# "" eval "" it is only for numbers 
n1=eval(input("enter value 1"))         
n2=eval(input("enter value 2"))
print(n1+n2)

enter value 1 20.6
enter value 2 10.3


30.900000000000002


In [52]:
print(type(n1))
print(type(n2))

<class 'float'>
<class 'float'>


In [53]:
#"" eval "" it is only for numbers 
n1=eval(input("enter value 1")) 
print(n1)

enter value 1 apple


NameError: name 'apple' is not defined

In [57]:
# 1 Home work
# take threee numbers a, b, c from the key board .
# fine sum and average


a= eval(input("Enter the a number"))
b= eval(input("Enter the a number"))
c= eval(input("Enter the a number"))
add=a+b+c
avg=add/3
print(f"addition of {a} {b} and {c} is",add)
print(f"average is",avg)

Enter the a number 10
Enter the a number 20
Enter the a number 10


addition of 10 20 and 10 is 40
average is 13.333333333333334


In [3]:
#2 calculate total salary fo the user    31/05/2026            from calss 12

basic_sal=eval(input("Enter salary:"))
DA=eval(input("enter DA amount:"))
HRA=eval(input("enter HRA amount:"))
total_sal=basic_sal+DA+HRA
print("total salary is:",total_sal)

Enter salary: 50000
enter DA amount: 20000
enter HRA amount: 10000


total salary is: 80000


In [7]:
# 3 calculate how much user is payingthe tax

basic_sal=eval(input("Enter salary:"))
tax=eval(input("enter the tax percentage"))
tax_paying=(basic_sal*tax)/100
print("total tax paying is:",tax_paying)

Enter salary: 100000
enter the tax percentage 10


total tax paying is: 10000.0


In [28]:
# 4 WAP ask the user get a value between 1 and 100
# perform the square of the number

#packege name :  random
# methid name :  randint


import random
num=random.randint(1,100)
print("random number is",num) 
square=num*num
print(f"square of the {num} is:",square)



random number is 5
square of the 5 is: 25


In [29]:
# 5 perform the square root of the random number number

import random
s=random.randint(1,100)
print("random number is ",s)
import math
print(f"square root of {s} is", math.sqrt(s))

random number is  93
square root of 93 is 9.643650760992955


In [31]:
# 6 taking input from the user
num=eval(input("enter the number"))
square=num*num
print(f"square of the {num} is:",square)


enter the number 5


square of the 5 is: 25


In [35]:
# 7 WAP ask the user to enter the distance
#    ask enter a rate per distance 
#   calculate the total charge

dis=eval(input("enter the distance in Km:"))
rate=eval(input("enter the rate per Km:"))
total_charge=dis*rate
print(f"total amount for {dis} km is {total_charge} rs")


enter the distance in Km: 100
enter the rate per Km: 15


total amount for 100 km is 1500 rs


In [37]:
# 8 ask the user  to enter total bill in doller 
# manager: dollers are not accepted
# user : what will accepted
# manager: only indian rupees 
# ask the user to enter one doller is equal to how muc rupees
# manager: thank you

dollers=eval(input("enter the bill in dollers $"))
print("manager: dollers are not acceptable")
print("user: what will accepted")
print("manager: only indian rupees")
rupees=eval(input("enter the one doller is equal to how much rupees"))
ind_rupees=dollers*rupees
print("indian rupees is",ind_rupees)


enter the bill in dollers $ 100


manager: dollers are not acceptable
user: what will accepted
manager: only indian rupees


enter the one doller is equal to how much rupees 85


indian rupees is 8500


In [38]:
# sentences should come after 2 ro 3 seconds so we will use impoer time for above code

import time
dollers=eval(input("enter the bill in dollers $"))
time.sleep(3)
print("manager: dollers are not acceptable")
time.sleep(3)
print("user: what will accepted")
time.sleep(3)
print("manager: only indian rupees")
time.sleep(3)
rupees=eval(input("enter the one doller is equal to how much rupees"))
time.sleep(3)
ind_rupees=dollers*rupees
print("indian rupees is",ind_rupees)

enter the bill in dollers $ 100


manager: dollers are not acceptable
user: what will accepted
manager: only indian rupees


enter the one doller is equal to how much rupees 85


indian rupees is 8500


In [ ]:
# father and son
# Father : which course you want to do
# S/D : I want to persue DS course
# Father: oh! that is very nice
# Father: May i know your JEE rank
# son : 20000
# Father: then how you can persue
# son: Payment seat
# Father: Which college will offer
# son: Naresh IT 
# Father: how much fee
# son: 20000
# Father: How many Sem
# son: 4 sem
# Father: how much it will cost
# son: 200000 per year


